dotnet-authserver/src/TeacherIdentity.AuthServer/Services/Csv/AzureUserImportCsvStorageService.cs

```diff
@@ -0,0 +1,26 @@
+using Azure.Storage.Blobs;
+
+namespace TeacherIdentity.AuthServer.Services.Csv;
+
+public class AzureUserImportCsvStorageService : IUserImportCsvStorageService
+{
+    private const string PendingFolderName = "pending";
+    private readonly string _connectionString;
+    private readonly string _userImportsContainerName;
+
+    public AzureUserImportCsvStorageService(IConfiguration configuration)
+    {
+        _connectionString = configuration.GetConnectionString("DataProtectionBlobStorage") ??
+           throw new Exception("Connection string DataProtectionBlobStorage is missing.");
+        _userImportsContainerName = configuration.GetValue("UserImportsContainerName", "user-imports") ?? "user-imports";
```

Comment:
Could we have an `Options` class here then bind to a `ConfigurationSection` instead of explicitly accessing `IConfiguration`?

Reply:
Done